src/components/MainLayout/RoutesNavigationList.tsx

```diff
@@ -19,7 +17,6 @@ export const RoutesNavigationList = ({ routes }: RoutesNavigationListProps) => {
           <NavLink
             to={route.path}
             className={`${!route.active ? 'ml-7' : ''} hyphens-auto text-button-md text-primary-gray hover:underline`.trim()}
-            lang={i18n.language}
```

Comment:
Was this unnecessary or why it was removed?

Reply:
unnecessary, I think